finding/finding.go

```diff
@@ -221,6 +221,13 @@ func (f *Finding) WithLocation(loc *Location) *Finding {
 	return f
 }
 
+// WithValues adds values to an existing finding.
+// No copy is made.
+func (f *Finding) WithValues(values map[string]int) *Finding {
+	f.Values = values
```

Comment:
The doc says "WithValues adds values to an existing finding", but the code replaces the values of a finding.

What do you expect if the code does this?
```

foo.WithValues(map[string]int{"foo": 1, "bar": 2})
foo.WithValues(map[string]int{"baz": 3, "qux": 4})
```

Should it have all 4 values or just the 2?

Reply:
+1, I copied the comment from another function.

Reply:
Had a chat with Laurent, we're fine with it either way in the future if the incremental approach is needed later.